Makefile

```diff
@@ -37,8 +37,19 @@ technical-docs:  ## build the technical docs
 guides:	swagger ## build the developer guide docs
 	cd docs/guides; make clean html
 
+# (IS_OPENEDX_TRANSLATIONS_WORKFLOW) is set to "yes" when openedx-translations repository extracts translations.
+# Related doc: https://docs.openedx.org/en/latest/developers/how-tos/enable-translations-new-repo.html
```

Comment:
it's not clear reading through this doc how it is related to this. I think it makes more sense to link to https://github.com/openedx/edx-platform/blob/59ddee7ad3363bf9f4bfae355fa3f24a1bbdfb37/docs/decisions/0018-standarize-django-po-files.rst once it is merged instead.